src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+function parseSalary(salary) {
+  return parseFloat(salary.replace('$', '').replace(',', ''))
+}
+
+function sortList(list) {
+  const items = Array.from(list.children);
+
+  items.sort((a, b) => {
+    const salaryA = parseSalary(a.dataset.salary);
+    const salaryB = parseSalary(b.dataset.salary);
+
+    return salaryB - salaryA;
+  });
+
+  items.forEach(item => list.appendChild(item));
+}
+
+function getEmployees(list) {
+  const employees = Array.from(list.children).map((name) => ({
```

Comment:
The variable `name` should be `item` to match the parameter in the map function. This will prevent a runtime error.